genrl/deep/agents/a2c/a2c.py

```diff
@@ -3,8 +3,8 @@
 import gym
 import numpy as np
 import torch
```

Comment:
Not needed. `torch` will get imported with `from torch` statement.

Reply:
Ok I didn't know that 😬

Reply:
This isn't happening. I still get an error saying `torch` isn't defined.

Reply:
EDIT: It does import torch but not `torch.Tensor` (used for typing hints). So either we just import torch or we use `from torch import Tensor` and use just `Tensor` which can be misunderstood. What do you think?